src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.searchBar {
+  display: flex;
+  position: absolute;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+  margin-left: 48px;
+}
+
+.searchBar--big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 17px;
+}
+
+.searchBar--small {
+  margin-left: 17px;
+  top: 90px;
+  height: 42px;
+}
+
+.searchBar__input {
+  width: 100%;
+  display: block;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px #3d4e611a;
+  font-style: normal;
+  border-radius: 3px;
+}
+
+.searchBar__input--top {
+  height: 70px;
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.searchBar--big:hover {
+  box-shadow: 0 4px 4px #00000040;
+}
+
+.searchBar--big:focus {
+  font-weight: 400;
+  background-color: #f6f6f7;
+}
+
+.searchBar--big:default {
+  box-shadow: 0 4px 4px #00000040;
+}
+
+.searchBar__input--bottom {
+  height: 42px;
+  background-position: 13px 14px;
+  background-size: 11px 11px;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.searchBar--small:default {
+  box-shadow: 0 4px;
+}
+
+.searchBar--small:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+  font-weight: 400;
+}
```

Comment:
The `:hover` pseudo-class should be applied to the interactive HTML element, not to its parent. In this case, it should be applied to the `input` element, not to the `div`. So it should be `.searchBar__input--bottom:hover` instead of `.searchBar--small:hover`.